client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -5,64 +5,71 @@
  */
 
 import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
-import { isObject } from 'lodash';
+import { flow, isObject } from 'lodash';
 
 /**
  * Internal dependencies
  */
+import { selectPostRevision } from 'state/posts/revisions/actions';
+import { isSingleUserSite } from 'state/sites/selectors';
 import PostTime from 'reader/post-time';
 
 class EditorRevisionsListItem extends PureComponent {
 	selectRevision = () => {
-		this.props.selectRevision( this.props.revision.id );
+		this.props.selectPostRevision( this.props.revision.id );
 	};
 
 	render() {
+		const { revision, isMultiUserSite, translate } = this.props;
 		return (
 			<button
 				className="editor-revisions-list__button"
 				onClick={ this.selectRevision }
 				type="button"
 			>
 				<span className="editor-revisions-list__date">
-					<PostTime date={ this.props.revision.date } />
-				</span>
-				&nbsp;
-				<span className="editor-revisions-list__author">
-					{ isObject( this.props.revision.author ) &&
-						this.props.translate( 'by %(author)s', {
-							args: { author: this.props.revision.author.display_name },
-						} ) }
+					<PostTime date={ revision.date } />
 				</span>
+
+				{ isMultiUserSite && (
+					<span className="editor-revisions-list__author">
+						{ isObject( revision.author ) &&
+							translate( '%(author)s', {
+								args: { author: revision.author.display_name },
+							} ) }
+					</span>
+				) }
+
 				<div className="editor-revisions-list__changes">
-					{ this.props.revision.changes.added > 0 && (
+					{ revision.changes.added > 0 && (
 						<span className="editor-revisions-list__additions">
-							{ this.props.translate( '%(changes)d word added', '%(changes)d words added', {
+							+
+							{ this.props.translate( '%(changes)d word', '%(changes)d words', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( '%(changes)d word removed', '%(changes)d words removed' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/post-revisions-popover)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).